src/createList.js

```diff
@@ -0,0 +1,50 @@
+import './style.css';
+
+const listContainer = document.querySelector('.list-container');
+// eslint-disable-next-line import/no-mutable-exports
```

Comment:
- [ ] OPTIONAL
You have **disabled linter check** on your code. Please enable **all** linter checks on your code to ensure that your code is robust code and clean.